conf/mini_provisioner/v2/ha.cleanup.tmpl.1-node

```diff
@@ -19,5 +19,8 @@ server_node:
       management:
         interfaces: TMPL_INTERFACES
         netmask: TMPL_NETMASK
+    site_id: TMPL_SITE_ID
+    rack_id: TMPL_RACK_ID
+    storageset_id: TMPL_STORAGESET_ID
```

Comment:
This should be storage_set_id. Can you please verify with conf store spec.

Reply:
I see it as storage_set_id in the old provisioner conf store file that I have.

Reply:
"storageset_id" changed to "storage_set_id".